test/integration/files.js

```diff
@@ -59,6 +60,47 @@ describe('files API', async function () {
     created.attributes.md5sum.should.equal('7Zfd8PaeeXsm5WJesf/KJw==')
   })
 
+  it('fails if the contentLength is too small', async function () {
+    const filename = 'foo_' + random()
+    const stream = new Readable()
+
+    stream.push('datastring1')
+    stream.push(null)
+
+    const created = await cozy.client.files.create(stream, {
+      name: filename,
+      contentType: 'application/json',
+      contentLength: 4
+    })
+
+    // created.should.be.rejectedWith(/content size/)
```

Comment:
why is it commented ?

Reply:
This was the expected behaviour but it actually ends up as a timeout

Reply:
Correction : The expected behaviour was that we got an error from the stack, but somewhere in the node-fetch / http.Request mechanism, the request actually get truncated.

Reply:
removed commented code